src/Form/Type/ProgrammeType.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Form\Type;
+
+use App\Entity\Programme;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class ProgrammeType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('name')
+            ->add('description')
+            ->add('startDate')
+            ->add('endDate')
+            ->add('isOnline')
+            ->add('maxParticipants')
+            ->add('trainer')
+            ->add('room')
+            ->add('customers')
```

Comment:
types here?

Reply:
Types are not mandatory as I have configured options to use entity types.... but sometimes are useful especially for formatting. I'll add. Most of the time default values are ok or I have not figured out what values are better.